src/AppBundle/Controller/PostController.php

```diff
@@ -30,4 +30,16 @@ public function onlPostViewAction($postId)
     {
 
     }
+
+    public function testJenkinsCI()
+    {
+        $string = 'string';
+
+        if ($string){
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: PEAR.ControlStructures.MultiLineCondition.SpaceBeforeOpenBrace
**Severity**: ERROR
**File**: /var/lib/jenkins/workspace/test2/src/AppBundle/Controller/PostController.php L38

There must be a single space between the closing parenthesis and the opening brace of a multi-line IF statement; found 0 spaces


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-390101337>*